android/app/src/main/python/mmrelay/plugins/mesh_relay_plugin.py

```
@@ -0,0 +1,218 @@
+            self.logger.debug(f"Skipping message from unmapped channel {channel}")
+            return False
+
+        packet_json = event.source["content"].get("meshtastic_packet")
+        if not packet_json:
+            self.logger.debug("Missing embedded packet")
+            return False
+
+        try:
+            packet = json.loads(packet_json)
+        except (json.JSONDecodeError, TypeError) as e:
+            self.logger.error(f"Error processing embedded packet: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)